githooks/install.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SCRIPT_DIR="$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")"
+REPO_DIR=$(git rev-parse --show-toplevel)
+REPO_DIR="$(readlink -f "${REPO_DIR}")"
+GIT_DIR="$REPO_DIR/.git"
+GIT_DIR="$(readlink -f "${GIT_DIR}")"
+
+if [[ ! -d "$GIT_DIR" ]]; then
+  if [[ -d "$REPO_DIR/../.git" ]]; then
+    GIT_DIR="$REPO_DIR/../.git"
+    GITHOOKS_PATH="$GIT_DIR/modules/sphinx-github-pages-generator/hooks"
+  else
```

Comment:
When can this happen? If we are in a submodule? Please add a comment

Reply:
The githooks are also copies from the old repository. I have a different issue in witch i will go through them in detail.